homeassistant/components/plugwise/climate.py

```diff
@@ -43,15 +46,20 @@
 DEFAULT_MIN_TEMP = 4
 DEFAULT_MAX_TEMP = 30
 
+# New CURRENT_HVAC mode
+CURRENT_HVAC_DHW = "dhw"
```

Comment:
This is now allowed. We norm this on climate.

Reply:
Sorry, did you mean to write "not allowed"? 
If yes, then I think you are saying: remove the dhw-related stuff, correct?

Reply:
DHW-mode has been removed from the code.

Reply:
Also, `if xy is not None` had been replace by `if xy` where possible.